apps/definition_dictionary/lib/dictionary/type/list.ex

```diff
@@ -48,6 +48,8 @@ defmodule Dictionary.Type.List do
   defimpl Dictionary.Type.Normalizer, for: __MODULE__ do
     alias Dictionary.Type.Normalizer
 
+    def normalize(_, nil), do: Ok.ok(nil)
```

Comment:
Should null list be transformed into empty list?

I'm guessing not since that could have other meanings? But it'd be nice to know lists are never null in the system. Just thinking aloud.

Reply:
I think we should keep writing is as `null` for now and look into if that's the behavior we want later. 

On smartcitiesdata, we had technical reasons to not `null` these. In hindsight, with our use of protocols, I'm not convinced we have any technical reason to do it anymore.

We _may_ want to do it for end users still, though.